tests/cbmc/include/cbmc_proof/endian.h

```diff
@@ -0,0 +1,52 @@
+/*
```

Comment:
Would this be useful in main s2n?

Reply:
:+1: this would clean up the implementation and make it more portable for BE systems.

Reply:
I'm gonna create a GitHub issue and post it here so we can track that.

Reply:
https://github.com/awslabs/s2n/issues/2030 to keep this suggestion on track.